### DISCUSSION TITLE
any special configuration when using jest and yup?

### DISCUSSION THREAD

**Discussion by lester-arevalo** [react-hook-form/react-hook-form#3832] [Q&A] [answered]
The unit test with RTL show the following:

SyntaxError: Cannot use import statement outside a module

      at Runtime.createScriptFromCode (../node_modules/jest-runtime/build/index.js:1350:14)
      at Object.<anonymous> (../node_modules/@hookform/resolvers/yup.js:1:1)

**Comment by bluebill1049** [comment]
can you share more code?

**Comment by lester-arevalo** [comment]
I have found examples of running a unit test with regular validations and no issues but seems with Yup some special adjustments are possibly needed in the jest.config file, but just thinking at loud...
`  const { handleSubmit, errors, reset, register, setValue } = useForm({
    mode: 'onBlur',
    defaultValues: values,
    resolver: yupResolver(validationsSchema),
  });`

**Reply by bluebill1049** [reply]
hmmm not sure we need that, we are running tests at the lib with yup.

**Comment by lester-arevalo** [comment, marked as answer]
copy .babelrc in babel.config.json solved it
